docs/usage_guide.rst

```diff
@@ -70,7 +70,7 @@ If you already have an account just write the name and the password at the sign
 Set up rotki Premium
 ======================
 
-If you decide to purchase `rotki Premium <https://rotki.com/products/>`_ at a later time, you can set it up via API Keys -> rotki Premium.
+If you decide to purchase `rotki Premium <https://rotki.com/products/>`_ at a later time, you can set it up via :menuselection:`API Keys --> rotki Premium`.
```

Comment:
what does the menuselection do here?

Reply:
It turns it into a breadcrumb.

## Example
From this:
<img width="694" alt="Screenshot 2022-05-27 at 09 23 16" src="https://user-images.githubusercontent.com/72208758/170661198-f284c413-64ac-4dab-9f2c-190aae66809a.png">

To this:
<img width="694" alt="Screenshot 2022-05-27 at 09 21 53" src="https://user-images.githubusercontent.com/72208758/170661156-2da54f7c-c6a8-4b3a-b4c0-c3f28e0f0c9e.png">

Reply:
ah very cool